app.js

```diff
@@ -0,0 +1,61 @@
+var AIchoice = ['rock', 'paper', 'scissors'];
+var rand = Math.floor(Math.random()*3);
+
+var AIscore = 0;
+var userScore = 0;
+
+var turns = 1;
+while(turns <= 5 && AIscore < 3 && userScore < 3){
+
+var userChoice = prompt('What do you choose?: (R)ock, (P)aper, (S)cissors');
+
+switch (userChoice) {
+  case 'R':
```

Comment:
This only accounts for if you put in an uppercase R, not particularly intuitive for the user.